projects/pt1/python/torch_mlir_e2e_test/test_suite/norm_like.py

```diff
@@ -489,3 +489,40 @@ def forward(self, x):
 def LayerNormNormalizeOverAllDimsModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(2, 2, 3))
 
+class InstanceNormModule(torch.nn.Module):
```

Comment:
Is this testing a case not tested by the other test below? If not, then we should remove it